engine/src/main/java/org/terasology/rendering/dag/RenderTaskListGenerator.java

```diff
@@ -15,41 +15,49 @@
  */
 package org.terasology.rendering.dag;
 
+import com.google.common.collect.Lists;
 import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * TODO: Add javadocs
  */
 public final class RenderTaskListGenerator {
+    private static final Logger logger = LoggerFactory.getLogger(RenderTaskListGenerator.class);
 
     public RenderTaskListGenerator() {
 
     }
 
     public List<RenderPipelineTask> generateFrom(List<Node> orderedNodes) {
-        List<RenderPipelineTask> taskList = com.google.common.collect.Lists.newArrayList();
+        logger.info("Task List: "); // TODO: remove in the future or turn it into debug
+        List<RenderPipelineTask> taskList = Lists.newArrayList();
         for (Node node : orderedNodes) {
-            // TODO: add "renderPipeline.addAll(node.getDesiredStateChanges())" here
+
```

Comment:
I thought that we might not need to have an  intermediate list. However I missed the fact that we eventually need to set default values along the way. Will be fixed! 👍